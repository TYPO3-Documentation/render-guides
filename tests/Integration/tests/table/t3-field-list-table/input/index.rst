==========================
t3-field-list-table tables
==========================

..  t3-field-list-table::
    :header-rows: 1

    -   :Header1:   Header1
        :Header2:   Header2

    -   :Header1:   1
        :Header2:   one

    -   :Header1:   2
        :Header2:   two
