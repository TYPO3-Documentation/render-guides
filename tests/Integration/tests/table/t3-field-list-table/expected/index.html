<!-- content start -->
        <div class="section" id="t3-field-list-table-tables">
    <h1>t3-field-list-table tables</h1>

            <div class="table-responsive">
    <table class="docutils align-default">
                            <thead>
            <tr>
                            <th><p>Header1</p></th>
                            <th><p>Header2</p></th>
                    </tr>
        </thead>
        <tbody>
    <tr class="row-odd">
            <td><p>1</p></td>
            <td><p>one</p></td>
    </tr>
    <tr class="row-even">
            <td><p>2</p></td>
            <td><p>two</p></td>
    </tr>
</tbody>
    </table>
</div>

    </div>

    <!-- content end -->
